packages/css/src/components/Form/form.css

```diff
@@ -122,3 +127,20 @@
 .form-field.icon-left.large .form-icon {
   left: var(--form-input-large-padding-x);
 }
+
+/* TEXTAREA */
+.form-label-wrap {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  line-height: 1.5;
+}
+
+textarea.form-input {
+  max-width: 100%;
```

Comment:
What's the effect of `max-width` here? `.form-input` already have `width: 100%` 

Shouldn't we set `min-width` to prevent things like:
<img width="403" alt="Screenshot 2023-01-27 at 13 37 40" src="https://user-images.githubusercontent.com/12156624/215077369-287e3de2-3489-4cba-a4b8-a14a3085c6ad.png">

Reply:
The max-width is to prevent dragging outside of the window/parent element. F.x.:
![textarea-max-width-example](https://user-images.githubusercontent.com/4611189/215083090-cafae18c-5591-458a-93cd-03a897637b02.png)